pkg/activator/handler/handler.go

```diff
@@ -128,8 +129,17 @@ func (a *activationHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		}, "ThrottlerTry")
 		trySpan.End()
 
-		if err == activatornet.ErrActivatorOverload {
-			http.Error(w, activatornet.ErrActivatorOverload.Error(), http.StatusServiceUnavailable)
+		var errmsg string
+		switch err {
+		case activatornet.ErrActivatorOverload:
+			errmsg = err.Error()
+			fallthrough
```

Comment:
Is fallthrough the right thing to do here? Do we want the other cases to execute as well?

Reply:
yep, tried to be clever, failed.